core/shared/src/test/scala/fs2/CompilationTest.scala

```diff
@@ -56,10 +61,17 @@ object ThisModuleShouldCompile {
 
   def polyId[F[_]: Applicative, A](stream: Stream[Id, A]): Stream[F, A] = stream.covaryId[F] through (_.take(2))
 
-  // With cats implicits enabled, some of the above fail to compile due to the cats syntax being invariant:
+  // Ensure that Stream#flatMap is favored over cats's flatMap
   {
     import cats.implicits._
-    Stream(1,2,3).covary[IO].flatMap(i => Stream.eval(IO.pure(i)))
-    (Stream(1,2,3).covary[IO].flatMap(i => Stream.eval(IO(i)))): Stream[IO,Int]
+    1 |+| 1 // Mask unused warning from cats.implicits._ import
+    Stream(1,2,3).flatMap(i => Stream.eval(IO.pure(i)))
+    (Stream(1,2,3).flatMap(i => Stream.eval(IO(i)))): Stream[IO,Int]
```

Comment:
👍 

I think we used to have another problem because of this conflict, but can't remember what :(

Reply:
Yeah, we'd pick up `FlatMapOps#flatMap` and it would complain b/c it wanted a `[B] Int => Stream[Pure, B]` but got `Int => Stream[IO, Int]`